src/tactic/squeeze.lean

```diff
@@ -148,8 +160,18 @@ do some s ← get_proof_state_after (tac ff (user_args ++ simp_args)),
    pure (user_args' ++ simp_args')
 
 /-- make a `simp_arg_type` that references the name given as an argument -/
+-- this is currently very badly written - using an `expr` constructor raw is not a good idea
 meta def name.to_simp_args (n : name) : tactic simp_arg_type :=
```

Comment:
Similarly, this no longer needs the `tactic` monad. That said, I'm confused about what it's returning. You're making a `local_const` with name `n` and type `#0`? Why? If the last argument is really a dummy variable, would it be cleaner to use `pexpr.mk_placeholder`? Or `default` if it really doesn't matter?

Reply:
I wanted `default` here, but `show pexpr, from default` doesn't seem to find the instance. I'll use `mk_placeholder`.

Reply:
The reason I make a `local_const` specifically is just because that's what I observed the `expr` type that you get from typing `simp_args` manually, and so if it works as a `local_const` it should work in the suggested `simp only` call.